gla11y.mk

```diff
@@ -10,9 +10,7 @@ a11y_verbose_1 =
 
 all-local: $(GLA11Y_OUTPUT)
 $(GLA11Y_OUTPUT): $(ui_files)
-	$(a11y_verbose) if [[ $(GLA11Y) == *gla11y ]]; then \
-	        python3 $(GLA11Y) -P $(srcdir)/ -f $(srcdir)/$(GLA11Y_FALSE) -s $(srcdir)/$(GLA11Y_SUPPR) -o $@ $(ui_files:%=$(srcdir)/%); \
-	        fi
+	$(GLA11Y) -P $(srcdir)/ -f $(srcdir)/$(GLA11Y_FALSE) -s $(srcdir)/$(GLA11Y_SUPPR) -o $@ $(ui_files:%=$(srcdir)/%)
```

Comment:
you probably shouldn't remove `$(a11y_verbose)`

Reply:
indeed, fixed